pom.xml

```diff
@@ -108,6 +108,30 @@
         <module>worker-tracking-report</module>
     </modules>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.hibernate</groupId>
+                <artifactId>hibernate-validator</artifactId>
+                <version>5.2.5.Final</version>
+            </dependency>
```

Comment:
I'm sure it's correct but as a matter of interest why does this one need to be specifically called out?

Reply:
There is a conflict between the versions between the two imported dependency management sections (caf-common(5.1.3.Final) and dropwizard-bom (5.4.2.Final)). If we use the version from either one of these the other no longer works. This version is the middle ground that works for both, I needed to specify it here so that it took precedence over the imported versions.